### PR TITLE
Reopening of #20797 Gold solidification

### PR DESCRIPTION
**Pull request by ma44**
:cl: ma44
add: You can now solidify liquid gold into sheets of gold with frostoil and a very little amount of iron
/:cl:

### CODE REVIEW

**Comment by Supermichael777** [comment]
chem turns power into trivializing mining

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 1]
```diff
@@ -52,12 +52,25 @@
...
 	var/location = get_turf(holder.my_atom)
 	for(var/i = 1, i <= created_volume, i++)
 		new /obj/item/stack/sheet/mineral/plasma(location)
 
+/datum/chemical_reaction/goldsolidification
+	name = "Solid Gold"
+	id = "solidgold"
+	required_reagents = list("frostoil" = 5, "gold" = 20)
+ 	mob_react = 1
+ 		new /obj/item/stack/sheet/mineral/gold(location)
```
Comment (Supermichael777) [inline code/modules/reagents/chemistry/recipes/others.dm]: wrong spot

**Comment by Tacolizard** [comment]
now make it so frost oil + dosh = cold hard cash

**Comment by XDTM** [comment]
Just saying, you can grab the 20% gold gene from a golden apple and the 25% frostoil gene from ice peppers, put them in a plant, raise to 100 potency, and watch as you get 10+ bars per harvest.

**Comment by Supermichael777** [comment]
The bartender has lime juice but not lemon juice whats up with that

**Comment by ExcessiveUseOfCobblestone** [comment]
ma44 buffing botany

**Comment by optimumtact** [comment]
How do you get liquid gold?

**Comment by XDTM** [comment]
Grind gold, golden apples

**Comment by optimumtact** [comment]
how easy are golden apples to make, and how much gold could you theoretically produce via this method

**Comment by XDTM** [comment]
They're the direct and only mutation of apples, meaning you need 50 mutagen on average to get it. It contains a 20% gold gene, which translates to 20 gold reagent at 100 potency. You can then transfer this (and frost oil) to a high yield, repeatedly harvestable plant, and the reaction will happen on harvest, dropping 10-20 gold bars at your feet. Plant several of these to get rich very, very fast. If you use ambrosia gaia too you can just drop a pair of trays at cargo and they'll be set for a lot of harvests.

**Comment by optimumtact** [comment]
We can't really accept this then because it will unbalance cargo again (I mean really you're going to make this PR and then appeal a botany ban and claim you're somehow improved?)

Turn this process into a machine that requires power, upfront minerals from mining and some levels of research (oh and a frame ordered from cargo)

**Comment by XDTM** [comment]
If you make the process more tedious it'll be far less abusable; for example, if you lower the gold in the gold gene or add a third chem for the reaction, you'll need to grind plants before mixing. This way, the process becomes mechanical enough that it requires full time investment by the botanist and is gated by the speed of grinding the plants.

**Comment by swindly** [comment]
Consider also that some gold starts in the vault and that cryo magic can replicate reagents.

**Comment by ma44** [comment]
Gold solidification (along with plasma solidification) cannot happen in a mob, so that cyro dupe trick will most likely not work.

**Comment by swindly** [comment]
Unlimited amounts of gold and frost oil can still be extracted by putting mobs in cryo and then butchering them, and the chef has more than a 90% chance of being able to produce frost oil without needing botany.

**Comment by PKPenguin321** [comment]
That would be tedious as fuck since you'd also need to grow monkeys, feed them the chems, cryo them, kill them, gib them, grind the meat. At that point I'd say it's enough work to just call it emergent

**Comment by ShadowDeath6** [comment]
If you go through so much work to where you turn monkey cubes into gold don't you deserve the fraction of the reagents you get from them?

**Comment by nullbear** [comment]
"Gold in vault" as a source for liquid gold? It's already gold. 

Botany is just imbalanced as it is.

**Comment by XDTM** [comment]
Fun fact: Meat can only hold 20 reagents, about 10 of which are taken by nutriment. At maximum upgrades the gibber can grab about 40 reagents total from a mob, and even then you need to grind each steak for the reagents. Don't bother doing this for chems you can get in any other way.

**Comment by optimumtact** [comment]
Alright let us see how this is

**Pull request merged**